resources/js/components/DashboardPage.vue

```diff
@@ -12,12 +12,6 @@
       </div>
       <DashboardAddData class="adddata justify-self-end" />
       <DashboardRightSidebar class="sidebar" />
-      <DiscourseDiscussion
```

Comment:
This component shouldn't be removed from the main dashboard.  (In this ticket, at least...).

Reply:
Agree, not sure why I zapped that.